catalog/monitors/aws-service-check.yaml

```diff
@@ -0,0 +1,31 @@
+# The official Datadog API documentation with available query parameters & alert types:
+# https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor
+
+aws-service-check:
+  name: "(AWS) System Maintenance"
+  type: service check
+  query: |
+    "aws.status".over("*").by("region","service").last(2).count_by_status()
+  message: |
+    ({stage} {region}) {instance_id} failed a status check
+  escalation_message: ""
+  tags:
+    - ManagedBy:Terraform
+  notify_no_data: false
+  notify_audit: true
+  require_full_window: true
```

Comment:
do we need `require_full_window: true` since it looks like the query does not use  any of the following

```
require_full_window a Boolean indicating whether this monitor needs a full window of data before it’s evaluated. We highly recommend you set this to False for sparse metrics, otherwise some evaluations are skipped. Default: True for “on average”, “at all times” and “in total” aggregation. False otherwise.
```

https://docs.datadoghq.com/monitors/guide/monitor_api_options/